src/dialog/reimu.c

```diff
@@ -64,11 +71,15 @@ void dialog_reimu_stage4(Dialog *d) {
 
 void dialog_reimu_stage4_post(Dialog *d) {
 	M(Left, "See, I don’t scare easily, so that didn’t work. You should have just told me your name.");
+	M(Right, "I didn't think it was possible, but you're so much stronger than before!");
+	M(Right, "Can't you remember that I'm Kurumi?");
+	M(Left, "You don't seem like the sort that is worth remembering after several years.");
+	M(Right, "Waah, so mean!  You were way cuter back then!");
 }
 
 void dialog_reimu_stage5(Dialog *d) {
 	M(Left, "I didn’t expect someone actually respectable like you to cause me trouble for no reason.");
-	M(Right, "You may be the Hakurei shrine maiden, but I’m afraid that you don’t have permission from Heaven to be here.");
+	M(Right, "You may be the Hakurei Shrine Maiden, but I’m afraid that you don’t have permission from Heaven to be here.");
 	M(Left, "What, are you saying that the gods suddenly don’t think I’m fit for duty anymore?");
 	M(Right, "No, that’s not it. We already have a more suitable candidate from our concerned Celestials, and she is due to arrive instead.");
```

Comment:
Not actually related to the changes, I'm just curious: who might that candidate be?

Reply:
Not really anyone specific since I don't think Tenshi would be a recommendation, but it's my idea that they're sending a competent celestial from her family (like one of the adults) to handle this issue.